docs/INSTALL.md

```diff
@@ -177,9 +178,15 @@ sudo apk add \
 #### macOS
 
 ```bash
-brew install cmake ninja qt@6
+brew install cmake ninja qt@6 git-lfs
```

Comment:
git-lfs is not required neither on Mac, nor on Linux or Windows. We do not use icu's lfs submodule.

What kind of issue did you get? After which steps? Is it really a blocker?

Reply:
I first did only a git clone of the main repo, but when doing the command to install the submodules I got the error that git-lfs was needed. I don't currently remember which one

After installing it, the git repo had become in an unstable state (couldn't continue the install of the submodules), so I had to redo it and did it with the right command that included the submodules from the start

Reply:
So... does following official instructions require lfs or not?

Reply:
Yes

I uninstalled git-lfs and tried again

```
% git clone --recurse-submodules https://github.com/organicmaps/organicmaps.git
````
...
```Cloning into '/Users/eivind/organicmaps/3party/fast_double_parser/benchmarks/dependencies/double-conversion'...
remote: Enumerating objects: 1489, done.        
remote: Counting objects: 100% (481/481), done.        
remote: Compressing objects: 100% (132/132), done.        
remote: Total 1489 (delta 407), reused 362 (delta 349), pack-reused 1008        
Receiving objects: 100% (1489/1489), 7.08 MiB | 11.55 MiB/s, done.
Resolving deltas: 100% (982/982), done.
Submodule path '3party/fast_double_parser/benchmarks/dependencies/abseil-cpp': checked out 'd936052d32a5b7ca08b0199a6724724aea432309'
Submodule path '3party/fast_double_parser/benchmarks/dependencies/double-conversion': checked out 'f4cb2384efa55dee0e6652f8674b05763441ab09'
Submodule path '3party/fast_obj': checked out '1a8060257a96401a9555a476bd13c3a87502c9b5'
Submodule path '3party/freetype/freetype': checked out '920c5502cc3ddda88f6c7d85ee834ac611bb11cc'
Submodule 'dlg' (https://github.com/nyorain/dlg.git) registered for path '3party/freetype/freetype/subprojects/dlg'
Cloning into '/Users/eivind/organicmaps/3party/freetype/freetype/subprojects/dlg'...
remote: Enumerating objects: 1180, done.        
remote: Counting objects: 100% (170/170), done.        
remote: Compressing objects: 100% (106/106), done.        
remote: Total 1180 (delta 84), reused 102 (delta 41), pack-reused 1010        
Receiving objects: 100% (1180/1180), 3.49 MiB | 9.20 MiB/s, done.
Resolving deltas: 100% (655/655), done.
Submodule path '3party/freetype/freetype/subprojects/dlg': checked out '72dfcc858c040c54a6a0b88fcb7e70ee186d3167'
Submodule path '3party/gflags': checked out 'a738fdf9338412f83ab3f26f31ac11ed3f3ec4bd'
Submodule path '3party/glm': checked out 'c834d06c2739f63e28e5665d4569dca9415e3891'
Submodule path '3party/googletest': checked out '58d77fa8070e8cec2dc1ed015d66b454c8d78850'
Submodule path '3party/harfbuzz/harfbuzz': checked out 'bc90b29b37fe3809f9e48aa7be08fbf2208e481a'
git-lfs filter-process: git-lfs: command not found
fatal: the remote end hung up unexpectedly
fatal: Unable to checkout '680f521746a3bd6a86f25f25ee50a62d88b489cf' in submodule path '3party/icu/icu'```

Reply:
git-lfs is not required to check out on my Mac and Ubuntu machines. What are the explanations?

@rtsisyk did you test it?

Reply:
@eisams It works on the Ubuntu. Could it be that you've enabled git-lfs support in git/in your git config system-wide somehow?

```
$ git-lfs
Command 'git-lfs' not found, but can be installed with:
sudo apt install git-lfs

$ git clone --recurse-submodules https://github.com/organicmaps/organicmaps.git
Cloning into 'organicmaps'...
remote: Enumerating objects: 605697, done.
remote: Counting objects: 100% (30075/30075), done.
remote: Compressing objects: 100% (2528/2528), done.
remote: Total 605697 (delta 28120), reused 28218 (delta 27176), pack-reused 575622
Receiving objects: 100% (605697/605697), 6.60 GiB | 23.50 MiB/s, done.
Resolving deltas: 100% (425424/425424), done.
Updating files: 100% (14167/14167), done.
Submodule '3party/Vulkan-Headers' 
[LONG LOG SKIPPED]
Submodule path 'tools/twine': checked out 'a9a97d19c51e11ff677168c983c86988b08c4c81'
```

Reply:
@rtsisyk I think this error appears due to some specific local environment setup. It does not reproduce on other machines, neither Mac, nor Linux. That is why it was absent in documentation.